.travis.yml

```diff
@@ -14,5 +14,5 @@ before_install:
 - travis_retry make deps
 
 script:
-- make check_license style unused test staticcheck check_assets
+- make check_license style unused test lint check_assets
```

Comment:
Why are we still using travis and circle? I thought there was some discussion to use only one of them?

Reply:
We hit memory limits when running the tests in Circle CI which is why Travis is still used.

Reply:
bummer :+1: